core/domain/exp_services_test.py

```diff
@@ -3129,6 +3129,10 @@ def test_draft_version_valid_false(self):
         self.assertFalse(exp_services.is_draft_version_valid(
             self.EXP_ID2, self.USER_ID))
 
+    def test_draft_version_no_draft_exists(self):
```

Comment:
This function name (and others) seems a bit cryptic. Do you mean test_draft_version_if_no_draft_exists?